app.js

```diff
@@ -1,249 +1,34 @@
-    await logger.info(sr);
-
-    await databaseSetup();
-
-    app.listen(PORT, "0.0.0.0", () => {
-      logger.info("HTTP server started...");
-      logger.info(`You can now access your server on port: ${PORT}`);
-      console.log(`You can now access your server on port: ${PORT}`);
-      if (typeof process.send === "function") {
-        // eslint-disable-next-line no-console
-        process.send("ready");
-      }
-    });
-  } catch (err) {
-    await logger.error(err);
-  }
-
-  // Routes
-  if (dbConnectionString === "") {
-    app.use("/", require("./server_src/routes/index", { page: "route" }));
-  } else {
-    try {
-      app.use("/", require("./server_src/routes/index", { page: "route" }));
-      app.use(
-        "/serverChecks",
-        require("./server_src/routes/serverChecks", { page: "route" })
-      );
-      app.use(
-        "/users",
-        require("./server_src/routes/users", { page: "route" })
-      );
-      app.use(
-        "/printers",
-        require("./server_src/routes/printers", { page: "route" })
-      );
-      app.use(
-        "/groups",
-        require("./server_src/routes/printerGroups", { page: "route" })
-      );
-      app.use(
-        "/settings",
-        require("./server_src/routes/settings", { page: "route" })
-      );
-      app.use(
-        "/printersInfo",
-        require("./server_src/routes/SSE-printersInfo", { page: "route" })
-      );
-      app.use(
-        "/dashboardInfo",
-        require("./server_src/routes/SSE-dashboard", { page: "route" })
-      );
-      app.use(
-        "/monitoringInfo",
-        require("./server_src/routes/SSE-monitoring", { page: "route" })
-      );
-      app.use(
-        "/filament",
-        require("./server_src/routes/filament", { page: "route" })
-      );
-      app.use(
-        "/history",
-        require("./server_src/routes/history", { page: "route" })
-      );
-      app.use(
-        "/scripts",
-        require("./server_src/routes/scripts", { page: "route" })
-      );
-      app.use(
-        "/input",
-        require("./server_src/routes/externalDataCollection", { page: "route" })
-      );
-      app.use(
-        "/client",
-        require("./server_src/routes/sorting", { page: "route" })
-      );
-      app.get("*", function(req, res) {
-        res.redirect("/");
-      });
-    } catch (e) {
-      await logger.error(e);
-    }
-  }
-};
-
-// Mongo Connect
 mongoose
-  .connect(dbConnectionString, {
+  .connect(process.env.MONGO, {
     useNewUrlParser: true,
     useUnifiedTopology: true,
-    useFindAndModify: false
-  })
-  .then(async () => {
-    //Check database actually works...
-    if (mongoose.connection.readyState !== 1) {
-      await databaseIssue();
-      let err = "No db connection...";
-      throw err;
-    }
+    useFindAndModify: false,
+    serverSelectionTimeoutMS: 2500,
   })
-  .then(() => setupServerSettings())
-  .then(() => serverStart())
+  .then(() => ensureSystemSettingsInitiated())
```

Comment:
This will trigger mongoose errors if not connected: we go into database fallback! Dont remove and dont merge together with the next then().